doc/specs/stdlib_linalg.md

```diff
@@ -206,3 +206,140 @@ program demo_outer_product
     !A = reshape([3., 6., 9., 4., 8., 12.], [3,2])
 end program demo_outer_product
 ```
+
+## `zeros/ones` - Create an vector or matrix of `integer/real/complex` type, given shape and `0/1` value.
+
+### Status
+
+Experimental
+
+### Class
+
+Pure function.
+
+### Description
+
+`zeros/ones` creates an vector or matrix of `integer/real/complex` type, given shape and `0/1` value.
+
+### Syntax
+
+For vector:  
+`result = [[stdlib_linalg(module):zeros(interface)]](dim)`  
+`result = [[stdlib_linalg(module):ones(interface)]](dim)`
+
+For matrix:  
+`result = [[stdlib_linalg(module):zeros(interface)]](dim1, dim2)`  
+`result = [[stdlib_linalg(module):ones(interface)]](dim1, dim2)`
+
+
+### Arguments
+
+`dim/dim1`: Shall be an `integer` type.
+This is an `intent(in)` argument.
+
+`dim2`: Shall be an `integer` type.
+This is an `intent(in)` argument.
+
+#### Note
+
+Because of `huge(integer :: i) == 2147483647`, the dimensional maximum length of array created by the `zeros/ones` function is `2147483647`. 
+
+### Return value
+
+Returns an `vector` or `matrix` of `integer` type, given shape and `0/1` value.
```

Comment:
```suggestion
Returns a `vector` or `matrix` of the given shape, filled completely with either `0` or `1` `integer` type values.
```
function doesn't take 0/1 as the input. Name of the function `zeros`/`ones` tells us which value should be used to fill.

Reply:
Thank you very much 💖, I will correct such writing errors or ambiguities next time.